tee-worker/ts-tests/stress/config.json

```diff
@@ -0,0 +1,11 @@
+{
+	"timerEpoch": 1692230400000,
+	"connections": 20,
+	"iterations": 10,
+	"substrateEndpoint": "ws://tee-staging.litentry.io:9944",
+	"workerEndpoint": "wss://tee-staging.litentry.io:2000",
```

Comment:
TODO: need a way to run this against local worker on CI. Config should prob. be a command line argument?